package.json

```diff
@@ -40,6 +39,7 @@
   "devDependencies": {
     "babel-core": "5.8.21",
     "babel-loader": "5.3.2",
+    "babel-relay-plugin": "^0.1.2",
```

Comment:
Oh yea, I was going to do this a while ago (or rather, I was going to make this a `file:` reference since it lives here and we weren't public).

This is good and will survive if we move that out of here at some point.